### ISSUE TITLE
don t work

### ISSUE THREAD

**Issue by mas7or123** [Kattrigerkatta/cv_eventlogger#1]
Hello i tryed the script and it dont work i tryed with Eulen and a normal job trigger and it does not work

**Comment by Kattrigerkatta** [comment]
> 
> 
> Hello i tryed the script and it dont work i tryed with Eulen and a normal job trigger and it does not work

What exactely does not work?

**Comment by Kattrigerkatta** [comment]
I uploaded a small fix

**Comment by Musiker15** [comment]
Yeah, I trigger `lester:vendita` from vangelico_robbery and `esx_dmvschool:addLicense` with Eulen Menu. Nothing showed up in Discord.

My Config:
`{
    event = 'esx_dmvschool:addLicense',
    whook = 'https://discord.com/api/webhooks/878997971247726643/A_xBpivNQXH9UAdSIC_Iq84pR2ANjMj1cRhzsQGV-jXyaRyGPsE8YdI4xbmZk5haz5uQ',
  },`
 ` {
    event = 'lester:vendita',
    whook = 'https://discord.com/api/webhooks/878997971247726643/A_xBpivNQXH9UAdSIC_Iq84pR2ANjMj1cRhzsQGV-jXyaRyGPsE8YdI4xbmZk5haz5uQ',
  },`
  `{
    event = 't1ger_bankrobbery:safeReward',
    whook = 'https://discord.com/api/webhooks/878997971247726643/A_xBpivNQXH9UAdSIC_Iq84pR2ANjMj1cRhzsQGV-jXyaRyGPsE8YdI4xbmZk5haz5uQ',
  },`

**Comment by Kattrigerkatta** [comment]
Are this server events? or client events?

**Comment by Musiker15** [comment]
> Are this server events? or client events?

This are server events
